src/features/subscription/mapSubscriptionThemeToName.ts

```diff
@@ -0,0 +1,10 @@
+import { SubscriptionTheme } from 'features/subscription/types'
+
+export const mapSubscriptionThemeToName: Record<SubscriptionTheme, string> = {
+  [SubscriptionTheme.CINEMA]: 'Cinéma',
+  [SubscriptionTheme.LECTURE]: 'Lecture',
+  [SubscriptionTheme.MUSIQUE]: 'Musique',
+  [SubscriptionTheme.SPECTACLES]: 'Spectacles',
+  [SubscriptionTheme.VISITES]: 'Visites et sorties',
+  [SubscriptionTheme.COURS]: 'Cours et Ateliers',
```

Comment:
D'après [le Figma](https://www.figma.com/file/l5YB1mtBc5696W07HgI4UJ/Abonnement?type=design&node-id=2510-27405&mode=design&t=eXdLBHEX64sambVx-4) :
```suggestion
  [SubscriptionTheme.COURS]: 'Activités créatives',
```
À voir si on veut modifier la clé aussi pour que ça reflète mieux ce que c'est.

Reply:
ça va pas poser soucis comme j'ai déjà fait des requêtes au backend ? (je pense que ça va casser mon utilisateur ahah)

Reply:
Tkt c'est juste testing, je peux réinitialiser ton user si tu veux (ou tu peux le supprimer à partir du [backoffice](https://backoffice.testing.passculture.team/admin/delete))